includes/fields.php

```diff
@@ -78,6 +78,12 @@ function pmpro_add_user_field( $where, $field ) {
  */
 function pmpro_add_field_group( $name, $label = NULL, $description = '', $order = NULL ) {
 	global $pmpro_field_groups;
+	//Bail if the group already exists.
+	foreach($pmpro_field_groups as $group) {
+		if($group->name == $name)
+		//already exists	
+		return false;
+	}
```

Comment:
```suggestion
	//Bail if the group already exists.
	foreach( $pmpro_field_groups as $group ) {
		if( $group->name === $name ) {
			//already exists
			return false;
		}
	}
```
WPCS and using === here. Both values should always be strings, but just in case.

Reply:
Thanks. Added the suggested change. Regarding  the questions 

> Do we know what the user would intend?
> When is this coming up most often?

From the issue itself it seems it comes from custom code added to functions.php file in the theme or snippet plugin. Actually, we deal with dups in our UI by adding a number between parenthesis. We may try that approach for dups coming from custom code as well, but also we may enforce users to adopt user fields from UI, i guess is something that comes from RH deprecation and should be an edge case and should tend to stop happening.